bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -85,7 +85,7 @@
               </ul>
             </div>
             {% if l10n_has_tag('firefox_new_other_platforms') %}
```

Comment:
> Yes, it's a very old string

Sorry what does this mean, that all locales have the modal strings translated? (there's more than just one that craig's pointing out here).